### ISSUE TITLE
Can't start hyperkube kube-apiserver: sql: unknown driver "sqlite3" (forgotten import?)

### ISSUE THREAD

**Issue by discordianfish** [k3s-io/k3s#206]
**Describe the bug**
Looks like the sqlite support doesn't work when running the kube-apiserver form the hyperkube binary:
```
# hyperkube kube-apiserver
I0309 16:08:28.515310   11402 serving.go:317] Generated self-signed cert (/var/run/kubernetes/apiserver.crt, /var/run/kubernetes/apiserver.key)
I0309 16:08:28.515458   11402 server.go:493] external host was not specified, using 192.168.0.2
I0309 16:08:28.516279   11402 server.go:147] Version: v1.13.4-k3s.1
I0309 16:08:32.317794   11402 plugins.go:158] Loaded 8 mutating admission controller(s) successfully in the following order: NamespaceLifecycle,LimitRanger,ServiceAccount,TaintNodesByCondition,Priority,DefaultTolerationSeconds,DefaultStorageClass,MutatingAdmissionWebhook.
I0309 16:08:32.317969   11402 plugins.go:161] Loaded 6 validating admission controller(s) successfully in the following order: LimitRanger,ServiceAccount,Priority,PersistentVolumeClaimResize,ValidatingAdmissionWebhook,ResourceQuota.
I0309 16:08:32.322176   11402 plugins.go:158] Loaded 8 mutating admission controller(s) successfully in the following order: NamespaceLifecycle,LimitRanger,ServiceAccount,TaintNodesByCondition,Priority,DefaultTolerationSeconds,DefaultStorageClass,MutatingAdmissionWebhook.
I0309 16:08:32.322233   11402 plugins.go:161] Loaded 6 validating admission controller(s) successfully in the following order: LimitRanger,ServiceAccount,Priority,PersistentVolumeClaimResize,ValidatingAdmissionWebhook,ResourceQuota.
F0309 16:08:32.334344   11402 storage_decorator.go:57] Unable to create storage backend: config (&{ /registry []    true 0xc0004937a0 <nil> 5m0s 1m0s}), err (sql: unknown driver "sqlite3" (forgotten import?))
```

**To Reproduce**
Just run `hyperkube kube-apiserver`

**Version**
0.2.0

**Comment by ibuildthecloud** [comment]
Can you test hyperkube from v0.1.0? I'll fix this in 0.2.0 but curious if it even worked before.

**Comment by brandond** [comment]
Closing due to age.
